mods/ra2/rules/soviet-infantry.yaml

```diff
@@ -92,12 +92,12 @@ shk:
 	Inherits: ^Infantry
 	Buildable:
 		Queue: Infantry
-		BuildAtProductionType: Infantry
```

Comment:
I looked to Flak Trooper and Conscript, they fooled me, i tought all don't have. OK i'll revert it.